### DISCUSSION TITLE
Ultra short @tu.ta email domain

### DISCUSSION THREAD

**Discussion by serrq** [tutao/tutanota#4930] [Ideas]
Sorry to have create a request here but I haven't a Reddit account (and I haven't any intention to own it).

I ask for a new ultra short Tutanota domain.
What do you think about it?

**Comment by ysfchn** [comment]
That could be cool, but there is no TLD named `.ta` exists according to [this list.](https://data.iana.org/TLD/tlds-alpha-by-domain.txt)
